test/parallel/test-event-emitter-listeners-side-effects.js

```diff
@@ -21,7 +21,6 @@ assert(fl.length === 1);
 assert(fl[0] === assert.fail);
 
 e.listeners('bar');
-assert(!e._events.hasOwnProperty('bar'));
```

Comment:
Oh good catch. Guess that makes it semver-major, We should probably use what you suggested to check it though.